Update structure of analyzeFace

renitreni opened the pull request:
I have refactored the analyzeFace function to improve user experience by showing a loader immediately and then running the analysis. This prevents the UI from freezing. I believe this addresses your request.

https://github.com/user-attachments/assets/22750fc1-bb76-4360-84e0-7e94762cf35d
(That is my son though. Haha!)

renier-30kft committed:
Update structure of analyzeFace

nitinyadav2188 commented:
I think you should set it to the center for better user experience 


thanks for telling, I thought he created it.......I was taken aback for a second 😮‍💨

renitreni committed:
Merge pull request #1 from nitinyadav2188/main

Get latest

renitreni committed:
I've implemented the requested changes to handle the centering behavior of the meme container and controls panel. The solution includes

renitreni commented:
Hi @nitinyadav2188 ,

- Added CSS classes for centering elements when they need to be centered
- Modified the JavaScript to dynamically adjust the grid layout:

  - When controls panel is hidden: The meme container is centered using full-width-center class and single column layout
  - When controls panel is shown: The normal two-column grid layout is restored

<img width="1502" height="737" alt="Screenshot 2025-10-12 at 8 29 54 AM" src="https://github.com/user-attachments/assets/b332963c-e2f8-4192-a335-14af66d3bdea" />
